app/controllers/dashboards_controller.rb

```diff
@@ -11,6 +9,8 @@ def search
     patients.each do |patient|
       @results << patient.pregnancies.most_recent
     end
+    @patient = Patient.new
```

Comment:
Since we're calling the _new_pregnancy partial from within the javascript now, the instance variables from the dashboard index weren't accessible. However, I can switch back to using #build instead of #new.